config/locales/en.yml

```diff
@@ -288,6 +288,18 @@ en:
       percentage_owned:
         input: What percentage share of their home does your client legally own?
         input_hint: Your client's name must be on the property deeds, lease or freehold.
+    property_entry:
```

Comment:
We seem to be adding this property_entry section twice - lines 291 and 408. But it's already present on line 279, so aren't both of these redundant?

Reply:
Managed to remove lots of unused translations using the 'test' setting which is just about to be merged